demisto_sdk/commands/common/tests/release_notes_test.py

```diff
@@ -1,91 +1,60 @@
-                    '\n+' \
-                    '\n ## [19.8.2] - 2019-08-22' \
-                    '\n   - Added the Virus Total permanent link to the context of the following commands: ' \
-                    '\n     - url' \
-                    '\n     - file' \
-                    '\n     - url-scan' \
-                    '\n     - file-scan' \
-                    '\n     - file-rescan' \
-                    '\n ## [19.8.0] - 2019-08-06' \
-                    '\n   - Updated outputs with new indicator fields.'
+FILES_PATH = os.path.normpath(os.path.join(__file__, f'{git_path()}/demisto_sdk/tests', 'test_files'))
+nothing_in_rn = ''
+rn_not_filled_out = '%%UPDATE_RN%%'
+rn_filled_out = 'This are sample release notes'
+diff_package = [(nothing_in_rn, False),
+                (rn_not_filled_out, False),
+                (rn_filled_out, True)]
 
-diff_added_line = 'diff --git a/Integrations/integration-VirusTotal_CHANGELOG.md b/Integrations/i' \
-                  'ntegration-VirusTotal_CHANGELOG.md\n' \
-                  '\nindex cbf564679..cffab9e90 100644' \
-                  '\n--- a/Integrations/integration-VirusTotal_CHANGELOG.md' \
-                  '\n+++ b/Integrations/integration-VirusTotal_CHANGELOG.md' \
-                  '\n@@ -1,15 +1,15 @@' \
-                  '\n ## [Unreleased]' \
-                  '\n-  - Added batch support for the **ip** and **url** and **domain** commands.' \
-                  '\n+  - Added batch support for the **ip** and **url** and **domain** commands1.' \
-                  '\n+ ' \
-                  '\n   - Fixed an issue where the DBotScore would create duplications in the incident context. ' \
-                  'This effects Demisto version 5.5 and higher.' \
-                  '\n ## [19.8.2] - 2019-08-22' \
-                  '\n   - Added the Virus Total permanent link to the context of the following commands: ' \
-                  '\n     - url' \
-                  '\n     - file' \
-                  '\n     - url-scan' \
-                  '\n     - file-scan' \
-                  '\n     - file-rescan' \
-                  '\n ## [19.8.0] - 2019-08-06' \
-                  '\n   - Updated outputs with new indicator fields.'
 
-diff_package = [(diff_nothing, False),
-                (diff_changed_the_same_line, False),
-                (diff_removed_line, False),
-                (diff_added_line, True)]
+@pytest.mark.parametrize('release_notes, expected_result', diff_package)
+def test_rn_master_diff(release_notes, expected_result, mocker):
+    """
+    Given
+    - Empty release notes.
+    - Not filled out release notes.
+    - Valid release notes
 
+    When
+    - Running validation on release notes.
 
-@pytest.mark.parametrize('release_notes_diff, expected_result', diff_package)
-def test_rn_master_diff(release_notes_diff, expected_result, mocker):
+    Then
+    - Ensure validation correctly identifies valid release notes.
```

Comment:
you need to writhe here what dose the validation expects to find/print in this case